<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainingForm</class>
 <widget class="QWidget" name="TrainingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel { background-color: beige; }</string>
       </property>
       <property name="text">
        <string>Раздел</string>
       </property>
       <property name="margin">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="childrenCollapsible">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QListWidget" name="listWidget">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QStackedWidget" name="stackedWidget">
        <widget class="QWidget" name="instructionPage">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>14</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel { background-color: lavender; }</string>
            </property>
            <property name="text">
             <string>Инструкция</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="margin">
             <number>6</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Вас приветствует программа &amp;quot;Тренажер&amp;quot; из комплекса &amp;quot;Offline наставник&amp;quot;!&lt;/p&gt;&lt;p&gt;Вы начали прохождение раздела, состоящего из нескольких кейсов. &lt;span style=&quot; font-weight:600;&quot;&gt;Кейс&lt;/span&gt; - это подробно описанная ситуация, в которой может оказаться педагог. В каждом кейсе вам будет предложено ответить на несколько вопросов, связанных с описанной ситуацией. Представьте, что бы вы сделали в данной ситуации.&lt;/p&gt;&lt;p&gt;Опубликовать свой ответ вы сможете при помощи кнопки &amp;quot;Ввести ответ&amp;quot;. Учтите, что после нажатия на данную кнопку &lt;span style=&quot; font-weight:600; text-decoration: underline;&quot;&gt;ответ больше нельзя будет изменить&lt;/span&gt;. После ввода ответа вам будет показан комментарий наставника - вариант ответа на вопрос, который предлагается составителем кейса.&lt;/p&gt;&lt;p&gt;Кейсы можно проходить &lt;span style=&quot; font-weight:600;&quot;&gt;в любом порядке&lt;/span&gt;, но чтобы закончить раздел, необходимо ввести ответ во всех кейсах. Вы можете переключаться между кейсами при помощи списка слева.&lt;/p&gt;&lt;p&gt;После завершения раздела появятся &amp;quot;Итоги раздела&amp;quot;, в которых будет указан статус отправки ваших ответов, а также представлено послесловие от составителя, список литературы и т.п. Если это возможно, &lt;span style=&quot; font-weight:600;&quot;&gt;программа отправит ваши ответы организатору обучения&lt;/span&gt; для анализа и, возможно, дальнейшего обсуждения с наставником. Если при отправке ответов произошла ошибка, либо требуется отправка ответов вручную, на странице &amp;quot;Итоги раздела&amp;quot; будет соответствующее сообщение об этом.&lt;/p&gt;&lt;p&gt;Если у вас возникли какие-то вопросы по использованию программы, обратитесь к организатору обучения.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="startButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Перейти к первому кейсу</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
